models/hymod/include/Hymod.h

```diff
@@ -4,6 +4,7 @@
 #include <cmath>
 #include <vector>
 #include "LinearReservoir.hpp"
+#include "Pdm03.h"
```

Comment:
What is Pdm03.h?  I don't see it as an addition in this PR nor is it in the source already.  Is it needed?  If so we need to stage it and maybe name it something a little more clear?

Reply:
Ok, I'll resolve this, but as soon as it is merged I will open an issue to clean up the Pdm03 so it is more clear, after looking at the file, may need to re-factor some of the internals as well.